src/three/plugins/GoogleCloudAuthPlugin.js

```diff
@@ -63,4 +77,38 @@ export class GoogleCloudAuthPlugin {
 
 	}
 
+	async fetchData( uri, options ) {
+
+		// wait for the token to refresh if loading
+		if ( this._tokenRefreshPromise !== null ) {
+
+			await this._tokenRefreshPromise;
```

Comment:
The session token will likely be modified by this point so we'll need to reassign it like below